### PR TITLE
cephfs-shell: drop LooseVersion for version.parse

### PR DESCRIPTION
**Pull request by joscollin**
Fixes: https://tracker.ceph.com/issues/62739

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins test make check arm64

**Comment by joscollin** [comment]
jenkins test make check arm64

**Comment by joscollin** [comment]
@vshankar Tests passed: http://pulpito.front.sepia.ceph.com/jcollin-2023-09-19_23:49:43-fs:shell-wip-B62739-cephfs-shell-distro-default-smithi/

**Comment by joscollin** [comment]
jenkins test make check arm64

**Comment by joscollin** [comment]
@vshankar ping, any review comments?

File: doc/man/8/cephfs-shell.rst [hunk 1]
```diff
@@ -56,7 +56,7 @@ Options
 
 .. code:: bash
 
-    [build]$ python3 -m venv venv && source venv/bin/activate && pip3 install cmd2
```
Comment (vshankar) [inline doc/man/8/cephfs-shell.rst:59]: This seems like a different fix. Although its in the docs, but does this need a separate PR?

Comment (joscollin) [reply doc/man/8/cephfs-shell.rst:59]: > This seems like a different fix. Although its in the docs, but does this need a separate PR?

This errored out without installing `colorama and packaging` while trying to start cephfs-shell. So I've updated the docs that didn't have them. Anyway, `packaging` is needed for this PR atleast.

Comment (vshankar) [reply doc/man/8/cephfs-shell.rst:59]: ack

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
https://pulpito.ceph.com/?branch=wip-vshankar-testing-20230926.081818

**Pull request merged**